package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "NODE_ENV=development nodemon -e coffee,jade,sass server.coffee",
-    "test": "./tests/run.coffee"
+    "test": "mocha --compilers coffee:coffee-script tests/tests/*"
   },
```

Comment:
i'm guessing we also want


    "devDependencies": {
      "mocha": "latest"
    }